deploy-mizar.sh

```diff
@@ -25,7 +25,7 @@ is_mizar_production=${1:-0}
 # Environments: k8s_kind, k8s_gcp, k8s_aws, 
 #               arktos_up, arktos_gcp, arktos_aws, 
 #               arktos_multi_master, arktos_with_kubemark, 
-mizar_environment=${2:-"arktos_up"}
```

Comment:
why can't we just use kind-setup.sh instead of deploy-mizar.sh

Reply:
deploy-mizar.sh can be considered as the only entrance of the deploy. It has parameter to decide whether it's arktos, or kind, or even other environment. It has parameter to decide whether it's dev or prod. For now we are using kind-setup for kind deploy. In the future, we should only keep one deploy entrance.